specifications/sep-payment-channel-mechanism.md

```diff
@@ -0,0 +1,657 @@
+[Change the Observation Period](#Change-the-Observation-Period) process.
+
+### Accounts
+
+The payment channel utilizes two Stellar accounts that are both 2-of-2
+multisig accounts while the channel is open:
+
+- E, the _escrow account_, that holds the assets that both participants have
+contributed to the channel and that will be distributed to the participants
+at channel close according to the final close transactions submitted.
+Created by I.  Jointly controlled by I and R while the channel is open.
+Control is returned to I at close.
+
+- V, the _reserve account_, that holds an amount of native asset contributed
+by R that will be used to sponsor the claimable balance ledger entries
+created at disbursement.  Created by R.  Jointly controlled by I and R while
+the channel is open.  Control is returned to R at close.  Cannot be merged
+until all claimable balances created at close for R are claimed by R.
+
+### Constants
+
+The two participants agree on the following constants:
+
+- m, the _maximum transaction count for an iteration's transaction set_, is
+defined as 2, the maximum number of transactions that can be signed in any
+process between the increments of iteration number i.
+
+### Variables
+
+The two participants maintain the following variables during the lifetime of
+the channel:
+
+- s, the _starting sequence number_, is initialized to one greater than the
+sequence number of escrow account E after E has been created. It is the first
+available sequence number for iterations to consume.
+
+- i, the _iteration number_, is initialized to zero.  It is incremented with
+every off-chain update of the payment channel state, or on-chain setup, deposit,
+withdrawal, etc.
+
+- e, the _executed iteration number_, is initialized to zero. It is incremented
+with every iteration submitted on-chain.
+
+### Computed Values
+
+The two participants frequently use the following computed values:
+
+- s_i, the _iteration sequence number_, is the sequence number that iteration
+i's transaction set starts at. Assuming the history of the payment channel has a
+single value for m it is computable as, s+(m*i).
+
+- s_e, the _executed iteration sequence number_, is the sequence number that the
+executed iteration e's transaction set starts at. Assuming the history of the
+payment channel has a single value for m it is computable as, s+(m*e).
+
+### Processes
+
+#### Setup
+
+To setup the payment channel:
+
+1. I creates the escrow account E.
+2. R creates the reserve account V.
+3. Set variable initial states:
+   - s to E's sequence number + 1.
+   - i to 0.
+   - e to 0.
+5. I and R build the formation transaction F.
+6. Increment i.
+7. Sign and exchange a closing transaction C_i, that closes the channel with
+disbursements matching the initial contributions.
+8. Sign and exchange a declaration transaction D_i.
+9. I and R sign and exchange signatures for formation transaction F.
```

Comment:
💡 this wasn't initially obvious to me, wondering if it' worth adding a note about it
```suggestion
9. I and R sign and exchange signatures for formation transaction F.
    - it's important F is signed/exchanged after C_i and D_i in the case I or R disappear
```

Reply:
Good call, I've added three new sections discussing this at relevant points in the doc. 86a02d0